bot.js

```diff
@@ -1,12 +1,21 @@
-// IIFE (Immediatly Invoked Function Expression) to set up variables
+const dotenv = require('dotenv');
+dotenv.config();
+
+
+// IIFE (Immediately Invoked Function Expression) to set up variables
 (function() {
 	global.Discord = require('discord.js')
 	global.fs = require('fs')
-	global.client = new Discord.Client()
+	// bot intents
+	myintents = new Discord.Intents(Discord.Intents.NON_PRIVILEGED)
+	myintents.add('GUILD_MEMBERS')
+	global.client = new Discord.Client({ws: {intents: myintents}})
+	global.invites = {}
```

Comment:
invites are going away

Reply:
i'll bring them back sometime later on, maybe in v2, but not now

Reply:
Right, I'll remove that.

Reply:
Changed in #73dd365; removed unnecessary invites line